code/modules/holomap/holomap.dm

```diff
@@ -0,0 +1,479 @@
+/*
+	Somone should totally add an action button to select a location when observing the map
+	and it display a list to select from (TGUI maybe!? color coded !? YES?)
+	then show an icon like you_are_here there
+*/
+
+///this will hold images rendered at roundstart
+GLOBAL_LIST_EMPTY(holomaps)
+
+/datum/holomap
+	var/obj/screen/holomap/using
+	var/obj/screen/you_are_here/arrow
```

Comment:
could you autodoc document both of these, it's likely later contributors to holomaps won't have a great grasp on what the code all does in here so it should be well documented.